extension/zpagesextension/config_test.go

```diff
@@ -37,5 +37,6 @@ func TestUnmarshalConfig(t *testing.T) {
 			ServerConfig: confighttp.ServerConfig{
 				Endpoint: "localhost:56888",
 			},
+			EnableExpvar: false,
```

Comment:
ineffectual, false is the default value fr `bool`s

Reply:
Fixed in another PR https://github.com/open-telemetry/opentelemetry-collector/pull/11964